e2e/compose-env.yaml

```diff
@@ -8,6 +8,8 @@ services:
       image: 'docker:${TEST_ENGINE_VERSION:-edge-dind}'
       privileged: true
       command: ['--insecure-registry=registry:5000']
+      depends_on:
+          - registry
```

Comment:
Please remove.     `depends_on` should not be necessary.

Reply:
Fixed